docs/framework/wpf/advanced/globalization-for-wpf.md

```diff
@@ -34,13 +30,13 @@ This topic introduces issues that you should be aware of when writing
  A character reference gives the number of the particular                          [!INCLUDE[TLA#tla_unicode](../../../../includes/tlasharptla-unicode-md.md)] character it represents, in either decimal or hexadecimal. The following example shows a decimal character reference.  
```

Comment:
Two nits unrelated to your PR:
nit #1: rather than "number", a more precise "UTF16 code unit"
nit #2: no one is likely to know what U+03E8 is, so I'm wondering if it's helpful to mention it. It's **COPTIC CAPITAL LETTER HORI**, or 'Ϩ'.